src/app/public/modules/inline-delete/inline-delete.component.spec.ts

```diff
@@ -0,0 +1,166 @@
+import {
+  async,
+  ComponentFixture,
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  expect,
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+import {
+  InlineDeleteTestComponent
+} from './fixtures/inline-delete.component.fixture';
+
+import {
+  SkyInlineDeleteFixturesModule
+} from './fixtures/inline-delete-fixtures.module';
+
+import {
+  SkyInlineDeleteType
+} from './inline-delete-type';
+
+describe('Inline delete component', () => {
+  let fixture: ComponentFixture<InlineDeleteTestComponent>;
+  let cmp: InlineDeleteTestComponent;
+  let el: HTMLElement;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        SkyInlineDeleteFixturesModule
+      ]
+    });
+
+    fixture = TestBed.createComponent(InlineDeleteTestComponent);
+    cmp = fixture.componentInstance;
+    el = fixture.nativeElement;
+  });
+
+  it('should emit the deleteTriggered event when the delete button is clicked', () => {
+    fixture.detectChanges();
+    const deleteTriggeredSpy = spyOn(cmp.inlineDelete.deleteTriggered, 'emit').and.callThrough();
+    (<HTMLElement>el.querySelector('.sky-btn-danger')).click();
+    fixture.detectChanges();
+    expect(deleteTriggeredSpy).toHaveBeenCalled();
+  });
+
+  it('should emit the cancelTriggered event when the cancel button is clicked', async(() => {
+    fixture.detectChanges();
+    const cancelTriggeredSpy = spyOn(cmp.inlineDelete.cancelTriggered, 'emit').and.callThrough();
+    (<HTMLElement>el.querySelector('.sky-btn-default')).click();
+    fixture.detectChanges();
+    fixture.whenStable().then(() => {
+      setTimeout(() => {
+        expect(cancelTriggeredSpy).toHaveBeenCalled();
+      }, 401);
```

Comment:
For future devs... would you mind parameterizing the `400` and the `600` below with a name that gives a little context? Something like 'timeForButtonToShow`, but like way better :)

Reply:
Done